pynicotine/gtkgui/search.py

```diff
@@ -300,10 +300,13 @@ class Search:
         ("archive", FileTypes.ARCHIVE)
     )
     FILTER_PRESETS = {
-        "filterbr": ("0", "128", "160", "192", "256", "320"),
-        "filtersize": (">10MiB", "<10MiB", "<5MiB", "<1MiB", ">0"),
-        "filtertype": ("audio", "image", "video", "text", "archive", "!executable", "audio|image|text")
+        "filterbr": ("!0", "128 <224", ">190 <=320", "=320 =1411", "320"),
```

Comment:
=1411 seems a bit too specific (and won't work for results from Nicotine+ clients until 3.3.0). I'd probably go with `=320` and `>320` for the last two presets.

Reply:
> `=320` and `>320` for the last two presets

Not sure about the last preset should be `320`?

Reply:
The idea for >320 was a lossless filter.

Reply:
I feel like the list of presets lacks an overall "high-quality" filter.

If one considers anything that is _320Kbps or above_ to be a minimum standard, then there is no present for that here.

Reply:
How about:
`        "filterbr": ("!0", "128 <=256", "192 <=320", "320 <=768", ">768"),`